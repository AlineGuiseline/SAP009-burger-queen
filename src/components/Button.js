import './Button.modules.css'

function Button(props){
    return(
        <div>
            <button className='btnEstilo'>{props.nome}</button>
        </div>
    )
}
export default Button