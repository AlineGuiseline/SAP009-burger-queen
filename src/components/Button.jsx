import React from 'react';
import styles from './Button.module.css';

function Button({ nome }) {
  return (
    <div>
      <button className={styles.btnEstilo} type="submit">{nome}</button>
    </div>
  );
}
export default Button;
