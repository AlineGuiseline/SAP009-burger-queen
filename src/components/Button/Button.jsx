import React from 'react';
import styles from './Button.module.css';

function Button(props) {
  return (
    <div>
      <button className={styles.btnEstilo} type="submit" onClick={props.onClick}>{props.children}</button>
    </div>
  );
}

export default Button;
