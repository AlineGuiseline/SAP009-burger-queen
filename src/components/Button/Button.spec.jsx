import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Button from './Button';

it('deve renderizar o botÃ£o corretamente', () => {
  render(
    <Button> Entrar </Button>,
  );
  const btn = screen.getByText('Entrar');
  expect(btn).toBeInTheDocument();
  expect(btn).toMatchSnapshot();
});
