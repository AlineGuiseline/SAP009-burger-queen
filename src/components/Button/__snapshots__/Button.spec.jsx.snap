// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deve renderizar o bot√£o corretamente 1`] = `
<button
  class="btnEstilo"
  type="submit"
>
   Entrar 
</button>
`;
